lightning/src/chain/channelmonitor.rs

```diff
@@ -1354,6 +1358,7 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitor<Signer> {
 					outpoint: OutPoint { txid: *txid, index: *index as u16 },
```

Comment:
You can make this into a separate variable and then log it below instead of `txid:index`.